pkg/resource/parameter_group/sdk.go

```diff
@@ -18,13 +18,15 @@ package parameter_group
 import (
 	"context"
 	"errors"
+	"fmt"
 	"reflect"
 	"strings"
 
 	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
 	ackcompare "github.com/aws-controllers-k8s/runtime/pkg/compare"
 	ackcondition "github.com/aws-controllers-k8s/runtime/pkg/condition"
 	ackerr "github.com/aws-controllers-k8s/runtime/pkg/errors"
+	ackrequeue "github.com/aws-controllers-k8s/runtime/pkg/requeue"
```

Comment:
This is a good example of how to seperate/order import paths

Reply:
Thank you for the example! Just a quick question. For a group of imports, shouldn't them be ordered alphabetically? Like putting ackcompare before ackv1alpha1?

Reply:
Good question! It's organized alphabetically using the import path not the import alias :)
For example in `ackcondition "github.com/aws-controllers-k8s/runtime/pkg/condition"`, `ackcondition` is the import alias and `"github.com/aws-controllers-k8s/runtime/pkg/condition"` is the import path.

Dev tip: You can use [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) with your favorite editor, so that it always formats the imports for you.

Reply:
Thank you for the explanation! However, I did check the doc and still feel a bit confused, because it only says adding a blank line between each group. For goimports, I tried to run "go install golang.org/x/tools/cmd/goimports@latest" in the terminal of GoLand but nothing happened.

Reply:
Should I just click "sort imports"? I saw the option of it.